static/js/components/ProgressWidget.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+
+export default class ProgressWidget extends React.Component {
+
+  props: {
+    actual: number,
+    total:  number
+  };
+
+  // this is temporary, we can remove it once it is integrate into learners dashboard
+  static defaultProps = {
+    actual: 3,
+    total:  5
+  };
+
+  render() {
+    const { actual, total } = this.props;
+
+    return (
+      <div className="progress-widget">
+        <div className="row-progress-widget">
+          <p className="text heading-text">Progress</p>
+        </div>
+        <div className="row-progress-widget">
+          <div className="circular-progress-widget">
+            {this.circularProgressWidget(80, 9, actual, total)}
+          </div>
+          <p className="text text-course-complete">Courses complete</p>
+          <p className="text heading-paragraph">
+            On completion, you can apply for <br/>
+            the Masters Degress Program</p>
+          <div className="apply-master-btn">
+             <RaisedButton
+               disabledBackgroundColor="#8ee0b0"
+               disabledLabelColor="#25b346"
+               label="Apply for Masters"
+               disabled={true} />
+          </div>
+        </div>
+      </div>
```

Comment:
This layout doesn't quite make sense to me. What do you think about doing this instead?

1. remove `display: flex` from `.progress-widget`
2. do `.progress-widget > div { width: 100%; }`

then the `<div>` elements immediately inside `progress-widget` will all be full-width (so they'll stack on their own). that would let you then set `display: flex` on your `.progress-widget-row` class, so then for the `header` you could do:

```css
.progress-widget-row {
  display: flex;
  flex-direction: row;

  &.header {
    justify-content: flex-start;
    align-items: center;
  }
}
```

then you could do something similar for the class that wraps the progress widget.

thoughts?